File: java/3-loop-struct/sum-of-even-numbers/src/Main.java
// Write a program that adds all even numbers up to 100. Print the sum.

public class Main {
    public static void main(String[] args) {

        int sum = 0;

        for (int i = 0; i <= 100; i++) {
            if (i % 2 == 0) {
                sum += i;
            }
        }

        System.out.println("The sum of even numbers is " + sum);
    }
}

File: c/3-loop-struct/sum-of-even-numbers/main.c
// Write a program that adds all even numbers up to 100. Print the sum.

#include <stdio.h>
#include <stdlib.h>

int main()
{
    int i, sum;

    for (i = 0; i <= 100; i++) {
        if (i % 2 == 0) {
            sum += i;
        }
    }

    printf("The sum of even numbers is %d.\n", sum);

    return 0;
}
